src/app/space/add-app-overlay/add-app-overlay.component.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  Component,
+  OnDestroy,
+  OnInit,
+  ViewEncapsulation
+} from '@angular/core';
+import { Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+
+import { Broadcaster } from 'ngx-base';
+import { Context, Space } from 'ngx-fabric8-wit';
+import { User, UserService } from 'ngx-login-client';
+
+import { ContextService } from 'app/shared/context.service';
+import { DependencyCheckService } from 'ngx-forge';
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'f8-add-app-overlay',
+  styleUrls: ['./add-app-overlay.component.less'],
+  templateUrl: './add-app-overlay.component.html'
+})
+export class AddAppOverlayComponent implements OnDestroy, OnInit {
+  currentSpace: Space;
+  loggedInUser: User;
+  projectName: string = '';
+  spaces: Space[] = [];
+  subscriptions: Subscription[] = [];
+
+  constructor(private context: ContextService,
+              private dependencyCheckService: DependencyCheckService,
+              private broadcaster: Broadcaster,
+              private userService: UserService,
+              private router: Router) {
+    this.subscriptions.push(userService.loggedInUser.subscribe(user => {
```

Comment:
This subscription isn't necessary any longer - please use currentLoggedInUser to avoid the sub